decidim-dev/lib/decidim/dev/assets/participatory_processes.json

```diff
@@ -77,8 +77,7 @@
         "ca": "<p>Et accusamus debitis. Sint voluptatem velit. Quia architecto at.</p>",
         "en": "<p>Sunt eius labore. Odit fugiat numquam. Et consequatur pariatur.</p>",
         "es": "<p>Rerum aperiam omnis. Optio est quo. Odit est eveniet.</p>"
-      },
-      "remote_hero_image_url": "http://localhost:3000/uploads/decidim/participatory_process_group/hero_image/1/city.jpeg"
```

Comment:
Why do you want to remove the file URLs from the test data if this is the bug you are fixing?

I understand that these are pointing to the old Carrierwave URLs but I believe you are stubbing these in the tests, right?